### DISCUSSION TITLE
php-http/message-factory

### DISCUSSION THREAD

**Discussion by vikingsurfeur** [mpdf/mpdf#1891] [General]
Hi there!

I hope everyone is well!
I just wanted to know if you plan to implement psr17/http-factory instead of php-http/message-factory soon.

Thanks for your work.

**Comment by finwe** [comment]
There is no https://packagist.org/search/?query=psr17%2Fhttp-factory package available. Please be more verbose about reasons why should this change be done.

**Reply by finwe** [reply]
There is [psr/http-factory](https://packagist.org/packages/psr/), but even version 1.0.0 requires PHP 7 which cannot be done sooner than mpdf 10.0 (with no ETA) because of PHP 5.6 support.

**Reply by vikingsurfeur** [reply]
Hi,

Exuce me for my quickly question.
Yep, this is psr/http-factory, my fault.
We want to move from php-http/message-factory because is no longer maintained.
I understand your prerequities.

Thank you fo your answer.

**Reply by finwe** [reply]
php-http/message-factory dependency was removed in #1863 which was released in the last version. Just upgrade to 8.1.6, the dependency was not used anywhere anyway.
